### ISSUE TITLE
df command to record space used

### ISSUE THREAD

**Issue by tim-moody** [iiab/iiab#3667]
I notice that `df -B1 --output=used /` is used in ? all roles to calculate space used.

1. It would be better practice to make this a variable rather than hard coding in more than a dozen places.
2. What happens if /library is a separate partition? Will anything installed there get included in the size, for example in osm or kalite?

**Comment by holta** [comment]
> 2. What happens if /library is a separate partition? Will anything installed there get included in the size, for example in osm or kalite?

Good point.  Indeed, an IIAB whose `/library` is on a separate partition will not have those bytes tabulated as is:
- If something like that is truly required, a command like `df -B1 --output=used /library` might help, for any directory within a partition (e.g. `/library` in this case you mention).
- Just FYI as background, `df --output[=FIELD_LIST]` accepts the fields below (according to `df --help`)

```
FIELD_LIST is a comma-separated list of columns to be included.  Valid
field names are: 'source', 'fstype', 'itotal', 'iused', 'iavail', 'ipcent',
'size', 'used', 'avail', 'pcent', 'file' and 'target' (see info page).
```

> 1. It would be better practice to make this a variable rather than hard coding in more than a dozen places.

I hear you.  And don't dismiss your perspective.  While personally I find code more readable without an excess of circuitous softcoding, i.e. with as much context as possible (actual shell commands in this case) visible very nearby.

In any case, for now it's 45 + 45 + 1 + 1 = 92 identical invocations of the command.  The 45 being IIAB's apps/services, and the 1+1 help tabulate overall disk footprint (i.e. `iiab_software_disk_usage = XXX` also /etc/iiab/iiab.ini) using:

- roles/0-init/tasks/create_iiab_ini.yml
- roles/9-local-addons/tasks/main.yml

Just FWIW being that all 92 are the same (for now!) changing them is certainly instant and easy (using global search/replace utilities like grep/sed/awk/etc).

Related:

- PR #3592
- PR #3597
